spec/core/finalization/schemas/FinalizationProofDTO.yml

```diff
@@ -0,0 +1,23 @@
+type: object
+required:
+  - version
+  - finalizationEpoch
+  - finalizationPoint
+  - height
+  - hash
+  - messageGroups
+properties:
```

Comment:
Don't you want to provide descriptions for these properties? They're all integers, so I cannot know what a Point or an Epoch is unless I read the docs.

Reply:
added some descriptions to the not so obvious fields, other than that i think it makes sense to read the docs for basic concepts, just like `transaction` wouldn't have a description ^^

if you think things should be different feel free to add the missing descriptions though

Reply:
I still don't know what's obvious and what's not in this project so I'll trust your judgement... for now 😈